app/addons/auth/base.js

```diff
@@ -19,58 +19,46 @@ Auth.session = new Auth.Session();
 FauxtonAPI.setSession(Auth.session);
 app.session = Auth.session;
 
-Auth.initialize = function () {
 
-  FauxtonAPI.addHeaderLink({
-    id: 'auth',
-    title: 'Login',
-    href: '#/login',
-    icon: 'fonticon-user',
-    bottomNav: true
-  });
+function cleanupAuthSection () {
+  FauxtonAPI.removeHeaderLink({ id: 'auth', bottomNav: true });
+}
+
+Auth.initialize = function () {
 
   Auth.session.on('change', function () {
-    var session = Auth.session;
-    var link = {};
+    const session = Auth.session;
 
     if (session.isAdminParty()) {
-      link = {
+      const link = {
         id: 'auth',
         title: 'Admin Party!',
         href: '#/createAdmin',
         icon: 'fonticon-user',
         bottomNav: true
       };
+
+      cleanupAuthSection();
+      FauxtonAPI.addHeaderLink(link);
+      FauxtonAPI.hideLogin();
+
     } else if (session.isLoggedIn()) {
-      link = {
+      const link = {
```

Comment:
This doesn't seem  correct, the `link` variable is declared twice and then used outside its scope